internal/session/update.go

```diff
@@ -74,12 +84,75 @@ func (u *sessionUpdater) run(ctx context.Context) error {
 					u.removePlayer(msg.ctx, s, msg.playerID)
 				case *updateMsgChangeStateTo:
 					u.changeStateTo(ctx, s, msg.nextState)
+				case *updateMsgUpdTaskAnswer:
+					u.updateAnswer(msg.ctx, s, msg.playerID, msg.answer, msg.ready, msg.taskIdx)
 				}
 			})
 		}
 	}
 }
 
+func (u *sessionUpdater) updateAnswer(
+	ctx context.Context,
+	s *UnsafeStorage,
+	playerID PlayerID,
+	answer TaskAnswer,
+	ready bool,
+	taskIdx int) {
+	state := s.sessionState(u.sid)
```

Comment:
Прошу не делать так. **_Очень_** вводит в заблуждение такое форматирование. Либо скобочку (`)`) на новую строку переноси, чтобы был контраст, либо сделай пустую строку между концом аргументов и телом функции. Где-то ещё такое было здесь в PR, тоже ввело в ступор.

```suggestion
	taskIdx int,
) {
	state := s.sessionState(u.sid)
```

или

```suggestion
	taskIdx int) {

	state := s.sessionState(u.sid)
```